build/mvn

```diff
@@ -48,11 +48,11 @@ install_app() {
     # check if we already have the tarball
     # check if we have curl installed
     # download application
-    [ ! -f "${local_tarball}" ] && [ -n "`which curl 2>/dev/null`" ] && \
+    [ ! -f "${local_tarball}" ] && [ -n "`type curl 2>/dev/null`" ] && \
```

Comment:
Why are we replacing `which` with `type`? What's the difference between the two commands?